.gitignore

```diff
@@ -119,6 +119,7 @@ packages/fxa-content-server/locale/*/*/*.po
 packages/fxa-content-server/rerun.txt
 packages/fxa-content-server/.tmp
 packages/fxa-content-server/.tscompiled
+packages/fxa-content-server/public/locales
```

Comment:
With `fxa-content-server`'s build dir being `dist` and seeing `.po` locales in  `i18n` and not seeing a standard `public` directory for it this feels slightly funky, but I think that l10n script copies them all in `public/locales` rather than root level `fxa-content-server-l10n` right and then they get copied over accordingly? Content-server staging seems fine from an l10n perspective so this also seems fine, just thinking outloud here. 

I think we could also remove the package-level gitignore references for `fxa-content-server-l10n` now but I don't mind those sticking around for a bit since the clones might still be hanging out locally. Maybe we can clean that up at the end of the l10n improvements epic.

Reply:
Correct, the l10n script copies them consistently into public/locales. Previously the files would end up in a repo called fxa-content-server-l10n, and then depending on the package might be copied again to public folder or possibly just copied into dist. TBH, I'd have to look back to get the exact details. What I do know is that there were differences from package to package.

I considered removing the `fxa-content-server-l10n` gitignore, but had the same though. It is probably better to give it some time since people might still have stray `fxa-content-server-l10n` folders. The extra ignore doesn't really hurt anything, IMO.